Zend/zend_language_scanner_defs.h

```diff
@@ -1,4 +1,4 @@
-/* Generated by re2c 0.13.5 */
+/* Generated by re2c 0.15.3 */
 #line 3 "Zend/zend_language_scanner_defs.h"
```

Comment:
You can probably just drop this file from the commit, as the only change is the comment?